core/src/tests/test-parachain/src/lib.rs

```diff
@@ -185,25 +189,26 @@ parameter_types! {
 
 impl cumulus_pallet_parachain_system::Config for Runtime {
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::RelayNumberStrictlyIncreases;
-	type DmpMessageHandler = DmpQueue;
+	type DmpQueue = EnqueueWithOrigin<(), sp_core::ConstU8<0>>;
```

Comment:
Why don't we use the recommended

```suggestion
	type DmpQueue = EnqueueWithOrigin<MessageQueue, AggregateMessageOrigin::Parent>;
```

Reply:
Updated this.